src/lib_windows.cr

```diff
@@ -122,6 +122,10 @@ lib LibWindows
     daylight_bias : Long
   end
 
+  TIME_ZONE_ID_UNKNOWN  = 0u32
+  TIME_ZONE_ID_STANDARD = 1u32
+  TIME_ZONE_ID_DAYLIGHT = 2u32
+
```

Comment:
Add `TIME_ZONE_ID_INVALID = -1.to_u32` as well, so that it can be used instead of `if ret == -1`